src/main/kotlin/kscript/app/ResolveIncludes.kt

```diff
@@ -37,20 +36,32 @@ private fun resolveIncludesInternal(template: File): File? {
             }
 
             try {
-                sb.appendln(includeURL.readText())
+                // collect the import or emit
+                includeURL.readText().lines().forEach {
+                    if (it.startsWith("import")) {
+                        imports.add(it)
+                    } else {
+                        sb.appendln(it)
+                    }
+                }
             } catch (e: FileNotFoundException) {
                 errorMsg("Failed to resolve //INCLUDE '${include}'")
                 System.err.println(e.message?.lines()!!.map { it.prependIndent("[ERROR] ") })
-
                 quit(1)
             }
+        } else if (it.startsWith("import")) {
+            imports.add(it)
         } else {
-            // if it's not a directive we simply skip emit the line as it is
-            sb.appendln(it)
+            // if its not an include directive or an import or a bang line, emit as is
+            if (it.startsWith("#")) {  } else { sb.appendln(it) }
```

Comment:
imho should be `else if(!it.startsWith("#!/"))` ( to get rid of the ugly `{ }` and to streamline the code